symFileManager.py

```diff
@@ -48,38 +50,46 @@ def GetLibSymbolMap(self, libName, breakpadId):
     if libName == "":
       return None
 
+    # Guess the name of the .sym file on disk
+    if libName[-4:] == ".pdb":
+      symFileName = re.sub(r"\.[^\.]+$", ".sym", libName)
+    else:
+      symFileName = libName + ".sym"
+
+    pathSuffix = os.path.join(libName, breakpadId, symFileName)
+    urlSuffix = "/".join([libName, breakpadId, symFileName])
+
     # Check cache first
     libSymbolMap = None
     if libName in self.sCache and breakpadId in self.sCache[libName]:
       libSymbolMap = self.sCache[libName][breakpadId]
       self.UpdateMruList(libName, breakpadId)
```

Comment:
so we really only need one LRU list. It will have the LRU info for the last let's say 1000 libraries and the disk-cache will have files for those 1000 libraries. The memory cache will only have in-memory version of the first 0 to 99 entries in the same LRU.